test/unit/orchestration/dhcp_test.rb

```diff
@@ -130,6 +130,40 @@ def teardown
     assert_equal 1, h.primary_interface.queue.items.select {|x| x.action.last == :del_dhcp }.size
   end
 
+  test "when an existing host trigger a 'rebuild', its dhcp record should be updated if no dhcp record is found" do
+    Net::DHCP::Record.any_instance.stubs(:valid?).returns(false)
+    h = FactoryGirl.create(:host, :with_dhcp_orchestration)
+
+    # We need that the medium used belongs to the operating system used
```

Comment:
Could you tell me lines numbers, it's not easy to see on the interface: it show lines 133 to 137, but the first is a start of test block, one is blank, another is a comment comment.